src/components/app/data/queries/utils.js

```diff
@@ -63,12 +63,12 @@ export async function getEnterpriseLearnerQueryData({
       matchedBFFQuery({ enterpriseSlug }),
     );
     enterpriseLearnerData = {
-      enterpriseCustomer: bffResponse.enterpriseCustomer,
-      activeEnterpriseCustomer: bffResponse.activeEnterpriseCustomer,
-      allLinkedEnterpriseCustomerUsers: bffResponse.allLinkedEnterpriseCustomerUsers,
-      staffEnterpriseCustomer: bffResponse.staffEnterpriseCustomer,
-      enterpriseFeatures: bffResponse.enterpriseFeatures,
-      shouldUpdateActiveEnterpriseCustomerUser: bffResponse.shouldUpdateActiveEnterpriseCustomerUser,
+      enterpriseCustomer: bffResponse.enterpriseCustomer || null,
+      activeEnterpriseCustomer: bffResponse.activeEnterpriseCustomer || null,
+      allLinkedEnterpriseCustomerUsers: bffResponse.allLinkedEnterpriseCustomerUsers || [],
+      staffEnterpriseCustomer: bffResponse.staffEnterpriseCustomer || null,
+      enterpriseFeatures: bffResponse.enterpriseFeatures || null,
+      shouldUpdateActiveEnterpriseCustomerUser: bffResponse.shouldUpdateActiveEnterpriseCustomerUser || null,
```

Comment:
I believe we should ensure the fallback values correspond to the TypeScript interface for `EnterpriseLearnerData` ([source](https://github.com/openedx/frontend-app-learner-portal-enterprise/blob/ff6a45a0290af69e53adfb0035e42e4145246343/src/types.d.ts#L63-L70)).

In the case of `enterpriseFeatures`, `null` is not specified; as such, it could be an empty object here.

In the case of `shouldUpdateActiveEnterpriseCustomerUser`, the type suggests it's always a boolean; as such, it should be `false` here.